bundles/org.eclipse.equinox.app/META-INF/MANIFEST.MF

```diff
@@ -2,29 +2,30 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.equinox.app; singleton:=true
-Bundle-Version: 1.6.100.qualifier
+Bundle-Version: 1.6.200.qualifier
```

Comment:
PDE's API-tolls complain if I bump the minor version that no new API has been added.
I can either stick with a micro-version bump (in contrast to the other changes) or I could just suppress the error and bump the minor version.

Reply:
It is funny that the tool complains that you raise the version more than the tool can ascertain. The tool can only know about syntactic differences which affect semantics. But we developers know more... :-)

Reply:
This technically would not be a minor version change because we stop exporting `org.osgi.service.application`, but then we do a reexport of a bundle that still does export the package.  So the overall package set provided is compatible with previous versions for the `equinox.app` bundle

Reply:
> It is funny that the tool complains that you raise the version more than the tool can ascertain. The tool can only know about syntactic differences which affect semantics. But we developers know more... :-)

Yep, it looks we developers still have to pay attention.


> This technically would not be a minor version change because we stop exporting `org.osgi.service.application`, but then we do a reexport of a bundle that still does export the package. So the overall package set provided is compatible with previous versions for the `equinox.app` bundle

Yes. Do you think we should enforce a minor-version bump here as well? I can simply suppress the warning/error with a corresponding additional setting.

Reply:
I'm resolving this conversation because we are no longer modifying app.